src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -5,6 +5,7 @@
 using System.Collections.Immutable;
 using System.Diagnostics;
 using System.Linq;
+using Microsoft.Cci;
```

Comment:
>Microsoft.Cci [](start = 6, length = 13)

Is this `using` needed?

Reply:
It's needed for `IMethodDefinition`. Would you prefer I remove the using and just make this `Cci.IMethodDefinition`? Or is there a workaround to access a similar class without the import?

Reply:
I see, thanks. CCI is used in emit only. We should use `IMethodSymbol` here.

---
In reply to: [307973639](https://github.com/dotnet/roslyn/pull/37381#discussion_r307973639) [](ancestors = 307973639)

Reply:
Thanks, will change.

Reply:
It looks like this can be removed now.

---
In reply to: [307973536](https://github.com/dotnet/roslyn/pull/37381#discussion_r307973536) [](ancestors = 307973536)